frame-server/server/profile/lib/businessRules.js

```diff
@@ -14,14 +14,14 @@ var businessRules = {};
    * Must be custom defined for each project.
    * @params {Object} server The current server instance.
    * @params {string} userid The desired user ID.
+   * @params {Object} userData The user's demographic data as passed from the browser.
    * @params {function} callback The callback function to pass the profile data to.
    * @returns {Object} The profile data object.
    */
 
-  businessRules.setProfileVariables = function (server, userid, callback) {
+   businessRules.setProfileVariables = function (server, userid, userData, callback) {
```

Comment:
@BlairCooper you are passing in user data but not using it. Do you intend to use it?

Reply:
Yes, in order for someone to access fields from demographics.json when setting profile variables userData needs to be passed in. Without userData, setProfileVaraibles doesn't have any context by which to set the profile variables.

In our implementation we will be examining the responses to fields from demographics.json to set profile variables.